Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -540,110 +465,3 @@ extension NewsTableViewController: NewsCellViewModelDelegate {
         performSegue(withIdentifier: "reaction", sender: viewModel)
     }
 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/b820878dd89b8764df1b985fe786295a0c1f0c46/Headlines/src/Controllers/NewsTableViewController.swift#L467"></span>File should contain 400 lines or less: currently contains 467<br />
<code>file_length</code> <code>NewsTableViewController.swift:467</code></td>
    </tr>
  </tbody>
</table>